templates/hubble/deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
             - name: "AUTH_TRUST_HOST"
               value: "true"
             - name: "NEXTAUTH_URL"
-              value: "https://{{ .Values.host }}/api/auth"
+              value: "{{ tpl .Values.nextauth_url_tpl . }}"
             - name: "LUNA"
```

Comment:
maybe dumb question on my end, but shouldn't this be just `.Values.nextauth_url_tpl`